### DISCUSSION TITLE
一直显示无法注册包

### DISCUSSION THREAD

**Discussion by 924103752** [ywmoyue/biliuwp-lite#921] [General]
<img width="744" alt="微信图片_20241122094906" src="https://github.com/user-attachments/assets/8b239f67-bd0d-4a9e-bbd3-eeb5122cae8e">

**Comment by ywmoyue** [comment]
@924103752  请问是什么修改版系统或者手动改过注册表吗

**Reply by 924103752** [reply]
确实系统问题，重装后好使了谢谢。

**Comment by ywmoyue** [comment]
在应用商店安装如 WinUI 2 Gallary这样的UWP应用能安装吗
